pkg/globalnet/controllers/iptables/iface.go

```diff
@@ -130,6 +130,10 @@ func (i *ipTables) GetKubeProxyClusterIPServiceChainName(service *corev1.Service
 	// CNIs that use kube-proxy with iptables for loadbalancing create an iptables chain for each service
 	// and incoming traffic to the clusterIP Service is directed into the respective chain.
 	// Reference: https://bit.ly/2OPhlwk
+	if len(service.Spec.Ports) == 0 {
```

Comment:
Ideally this should never happen because we cannot create a Service without ports in K8s. 

I think we should rather fix the Headless service definition to include the following 
https://github.com/submariner-io/submariner/blob/b444546153c539d19effb3a477c45f0653b0f946/pkg/globalnet/controllers/controllers_suite_test.go#L428

```
			Ports: []corev1.ServicePort{{
				Name:       serviceName,
				Port:       int32(8080),
				TargetPort: intstr.FromInt(8080),
				Protocol:   corev1.ProtocolTCP,
			}},
```